src/app/api/v1/logout.py

```
@@ -33,12 +31,14 @@ async def logout(
-        raise HTTPException(status_code=500, detail=str(e))
+    async with httpx.AsyncClient() as client:
+        try:
+            response = await client.post(
+                f"{DOCKER_SERVICE_URL}/containers/stop/{agent_id}"
+            )
+            response.raise_for_status()
+            return {"message": f"Agentic app {agent_id} stopped successfully"}
+        except httpx.HTTPError as e:
+            raise HTTPException(
+                status_code=500, detail=f"Docker service error: {str(e)}"
+            )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)